### PR TITLE
 cat class add

### PR DESCRIPTION
**Pull request by leosilva12**

### CODE REVIEW

File: src/Main.java [hunk 1]
```diff
@@ -3,6 +3,8 @@ public class Main {
     public static void main(String[] args) {
         Animal a1 = new Dog();
         a1.makeSound();
+        Animal a2 = new Cat();
```
Comment (gnievestm) [inline src/Main.java]: Can you please use a descriptive name for the variable, maybe something like `Animal cat= new Cat()?`

File: src/Cat.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class Cat extends Animal {
+    @Override
+    public void makeSound() {
+        System.out.println("The cat barks: miauuuu!");
```
Comment (gnievestm) [inline src/Cat.java]: The cat should `meows: "miauuuu!"`

File: src/Main.java (hunk 1 above)
Comment (leosilva12) [reply src/Main.java]: ready, I have changed the name of the variables

File: src/Cat.java (hunk 2 above)
Comment (leosilva12) [reply src/Cat.java]: ready, I have changed the name of the variables

**Pull request merged**